garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -308,4 +308,4 @@ function PANEL:DoRightClick()
    menu:Open()
 end
 
-vgui.Register( "TTTScorePlayerRow", PANEL, "Button" )
+vgui.Register( "TTTScorePlayerRow", PANEL, "DButton" )

```

Comment:
Actually, this change doesn't really matter.
[Button](http://wiki.garrysmod.com/page/Category:Button) is deprecated. Changing this does not make a big difference. But I had the time and making a new PR for something like this doesn't really make sense.